tools/WixRelease.relproj

```diff
@@ -61,6 +61,12 @@
         <Title>WiX v$(MajorMinorVersionDottedString) source and symbols for debugging purposes.</Title>
         <Protected>true</Protected>
       </ReleaseFile>
+      
+      <!-- Upload the Nuget package to the website, at least until it gets published to nuget.org -->
+      <ReleaseFile Include="$(OutputPath)WixToolset.Tools.$(FullBuildVersionString).nupkg">
+        <Title>WiX v$(MajorMinorVersionDottedString) WixToolset.Tools Nuget package.</Title>
+        <Protected>true</Protected>
+      </ReleaseFile>
```

Comment:
What's the plan for the nuget package(s) from weekly builds?  Will they be made available on Nuget.org?

What if the Nuget package is uploaded on our storage like the msi/cab files (hidden)?  Seems like a bad idea for the build to push to Nuget.org and then we don't keep it anywhere.